README.md

```diff
@@ -25,6 +25,28 @@ Some experience in Scala. Some familiarity with big data or parallel processing
 * Overview Spark Streaming
 * Spark Architecture and Cluster Deployment
 
+## How this repo is organized
+
+* [/data](./data): contains some files that will be used along the workshop as a sample data.
+* [/dist](./dist): binaries and config files that would be useful to setup the Spark cluster locally. We'll see how to use them in the next section.
+* [/labs](./labs): contains all the commands used along the workshop with some details.
+* [/sampleapp](./sampleapp): scala application used to package and submit to the Spark Cluster with _spark-submit_ command.
+* [/sbt](./sbt): sbt binaries, useful to work with the *sampleapp* scala application.
+* [/scripts](./scripts): contains the shell script to run the Spark Cluster locally, initially with a Spark Master and two Spark Worker nodes.
+
+## Setup Locally
+
+* Install [Oracle Java 7/8](https://www.oracle.com/java/index.html) in case you haven't yet.
+* Download Spark binary distribution. In this workshop we'll use [Spark 1.5.1 Pre-built for Hadoop 2.4](http://d3kbcqa49mib13.cloudfront.net/spark-1.5.1-bin-hadoop2.4.tgz).
+* Uncompress the file in your local machine and set up the **SPARK_HOME** environment variable, with the whole path. For instance:
+    `export SPARK_HOME=/path/to/spark-1.5.1-bin-hadoop2.4`
+* File [/data/spark-env.sh](./dist/spark-env.sh) must be copied to `$SPARK_HOME/spark-env.sh`.
+* Let's run the Spark Cluster and the Spark Shell, altogether into a single step:
+    `sh scripts/sparkws.sh`
+* That's all, if everything was fine you should be able to see the local Spark Standalone Deployment into your browser: http://localhost:8080 .
+
+![" "](http://jpedrom.es/1PuYcKF "Spark Standalone Deployment")
```

Comment:
Please use a company domain for this image or upload directly to the repo on github